jwcrypto/jwe.py

```diff
@@ -635,6 +701,20 @@ def serialize(self, compact=False):
                 rec = self.objects['recipients'][0]
             else:
                 rec = self.objects
+            if 'header' in rec:
```

Comment:
I mean, can't you pawn this off on the caller? ISTM that placing iv/tag in the recipient header is the right move. This is what I do in jose...